### PR TITLE
Fix Code Quality

### PR DESCRIPTION
**Pull request by jamessinmaojun**

### CODE REVIEW

**Review by chiageng — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/RedoCommand.java [hunk 1]
```diff
@@ -28,7 +29,7 @@ public RedoCommand() {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        logger.log(Level.INFO, "Going to execute redo command.");
+        logger.log(Level.INFO, LOGGER_MESSAGE);
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/commands/RedoCommand.java:32]: should it be more specific, like LOGGER_REDO_MESSAGE?

**Review by yleeyilin — COMMENTED** [review]
minor comment

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -47,9 +48,9 @@ public class AddressBookParser {
     /**
      * Parses user input into command for execution.
      *
-     * @param userInput full user input string
-     * @return the command based on the user input
-     * @throws ParseException if the user input does not conform the expected format
+     * @param userInput The full user input string.
+     * @return the command based on the user input.
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:52]: Should be capital letter "The command based on the user input."

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 3]
```diff
@@ -134,9 +135,8 @@ public Command parseCommand(String userInput) throws ParseException {
             return new SortCommandParser().parse(arguments);
 
         default:
-            logger.finer("This user input caused a ParseException: " + userInput);
+            logger.finer(MESSAGE_PARSEEXCEPTION + userInput);
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:138]: MESSAGE_PARSE_EXCEPTION

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java [hunk 4]
```diff
@@ -42,7 +46,7 @@ public DeleteCommand parse(String args) throws ParseException {
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
         boolean isPreambleEmpty = argMultimap.getPreamble().isEmpty();
         if (!isPreambleEmpty) {
-            logger.log(Level.WARNING, "Parsing error while parsing for delete command.");
+            logger.log(Level.WARNING, MESSAGE_PARSEEXCEPTION);
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/DeleteCommandParser.java:49]: Same here

File: src/main/java/seedu/address/logic/parser/NoteCommandParser.java [hunk 5]
```diff
@@ -44,7 +48,7 @@ public NoteCommand parse(String args) throws ParseException {
                 FAILED_TO_ADD_NOTE, PREFIX_NAME, PREFIX_NOTE);
         boolean isPreambleEmpty = argMultimap.isPreambleEmpty();
         if (!isPreambleEmpty) {
-            logger.log(Level.WARNING, "Parsing error while parsing for note command.");
+            logger.log(Level.WARNING, MESSAGE_PARSEEXCEPTION);
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/NoteCommandParser.java:51]: same here

**Review by yleeyilin — COMMENTED** [review]
Left some comments!

**Review by jannaleong — APPROVED** [review]
lgtm

File: src/main/java/seedu/address/logic/commands/RedoCommand.java (hunk 1 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/commands/RedoCommand.java:32]: resolved

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 2 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:52]: resolved

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 3 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:138]: resolved

File: src/main/java/seedu/address/logic/parser/DeleteCommandParser.java (hunk 4 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/parser/DeleteCommandParser.java:49]: resolved

File: src/main/java/seedu/address/logic/parser/NoteCommandParser.java (hunk 5 above)
Comment (jamessinmaojun) [reply src/main/java/seedu/address/logic/parser/NoteCommandParser.java:51]: resolved

**Review by yleeyilin — APPROVED** [review]
lgtm

**Review by Joshy837 — APPROVED** [review]
LGTM

**Pull request merged**